src/main/java/org/prebid/server/analytics/reporter/greenbids/model/GreenbidsBid.java

```diff
@@ -19,16 +22,23 @@ public class GreenbidsBids {
     @JsonProperty("hasBid")
     Boolean hasBid;
 
-    public static GreenbidsBids ofBid(String seat, Bid bid) {
-        return GreenbidsBids.builder()
+    JsonNode params;
+
+    BigDecimal cpm;
+
+    String currency;
+
+    public static GreenbidsBid ofBid(String seat, Bid bid) {
+        return GreenbidsBid.builder()
                 .bidder(seat)
                 .isTimeout(false)
                 .hasBid(bid != null)
+                .cpm(bid.getPrice())
                 .build();
     }
```

Comment:
todo: as said by AntoxaAntoxic, here you build a `GreenbidsBid` while it is not complete (it misses the `params` and the `currency`.
So return the builder instead, and let the caller call the `build()` action.

Reply:
fixed